doc/contributing/README.md

```diff
@@ -1,38 +1,27 @@
-# Contributing to Bundler
+# Bundler Contributor Guidelines
 
-Welcome to Bundler! We are so happy that you're here. We know joining a new open source project can be daunting, so here's a quick overview of what you can expect from this documentation.
+Thank you for your interest in making Bundler better! We welcome contributions from everyone. Dozens of contributors like you have submitted feature improvements, fixed bugs, and written new documentation. [Join the Bundler Slack community](http://slack.bundler.io/) to connect with the Bundler core team and other contributors like you.
 
-*Something missing?* Send us a [pull request](../development/PULL_REQUESTS.md)!
+Before submitting a contribution, please take a moment to read through the following guidelines:
 
-**Recommended first steps**
+* [Bundler Code of Conduct](https://github.com/bundler/bundler/blob/master/CODE_OF_CONDUCT.md) (By participating in Bundler, you agree to abide by the terms laid out in the CoC.)
+* [Development Setup Guidelines](https://github.com/bundler/bundler/blob/master/doc/development/SETUP.md)
+* [Issue Reporting Guidelines](https://github.com/bundler/bundler/blob/master/doc/contributing/ISSUES.md)
+* [Pull Request Guidelines](https://github.com/bundler/bundler/blob/master/doc/development/PULL_REQUESTS.md)
 
-- Join us in the Bundler slack! Generate an invite [here](http://slack.bundler.io/)
-- *If you're interested in helping with code:*
-  - Get a quick overview of our [development process](../development/README.md)
-  - [Setup your machine for development](../development/SETUP.md)
-  - Checkout [How you can help: your first contributions!](HOW_YOU_CAN_HELP.md) for a list of suggestions to get started
-- *If you're interested in helping with documentation:*
-  - Read up on [how documentation works for Bundler](../documentation/README.md)
-  - Learn about our [documentation vision](../documentation/VISION.md)
 
-You can start learning about Bundler by reading [the documentation](http://bundler.io). If you want, you can also read a (lengthy) explanation of [why Bundler exists and what it does](http://bundler.io/rationale.html).
+## Contributing to Bundler
 
-## [How you can help: your first contributions!](HOW_YOU_CAN_HELP.md)
+Now that you’ve looked over our contribution guidelines, here are the different ways you can start contributing to the Bundler project:
```

Comment:
I think you mean contributor guidelines. However, I'd leave out the first half of the sentence, "Now that.... guidelines" to keep it simple.